summarization/summarizer_main.py

```diff
@@ -210,3 +222,12 @@ def _assemble_and_instantiate_stamp_page(
             stamp_position,
             stamp_descriptor_embedding
         )
+
+    def _fetch_and_set_stamp_descriptors_dict_for_embedded_content(self):
+        self.embedded_descriptors_dict = dict()
+        file_path = 'summarization/stamp_descriptors_for_embedded_content.json'
+        with open(file_path, 'r') as file:
+            self.embedded_descriptors_dict = json.load(file)
+
+    def _get_stamp_descriptor_for_embedded_content(self, content):
+        return self.embedded_descriptors_dict[str(content.content_type)]
```

Comment:
Do we need string conversion here? Can we use content.content_type directly?

Reply:
Yes! dict keys can only be of type int,str,float etc . It threw an error saying it can't be of type ContentType.<TypeName> when i tried that.

Reply:
Could you elaborate on this? ContentType.TEXT (say) is an int right? So you should be able to do d[ContentType.TEXT] = "".
Am I missing something here?

Reply:
1) The enum entity ContentType.TEXT (say) is a tuple of (name,value) - so we could use ContentType.TEXT,value
2) The problem with the above would be that the python dict would store an int as the key - but these would be converted to string when dumping and loading with json - so we would have to anyways type cast the key obtained in the content.content_type to a string before using it as a python key.

Regardless, i've updated it to use only the .value attribute for clarity! Hope this is okay.

Reply:
Let's keep it as is then (without .value).

Reply:
Okay!